src/components/routing/Navigation/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+
+function CustomLink({ children, to }) {
+  const match = useRouteMatch({
+    path: to,
+    exact: true,
+  });
+  return (
+    <Link className={match ? 'link-selected' : 'link'} to={to}>
```

Comment:
It's so impressive! 😮 `match` is so useful to know what the current route is. Nice!